src/services/useFeedbackService.ts

```diff
@@ -25,7 +25,7 @@ const useFeedbackService = () => {
 						email: email
 					})
 				});
-				return await response.json();
+				return response;
```

Comment:
How is this response used?
You would want to handle errors if the response.status is not 200.

Reply:
basically I was encountering a weird bug where 400 or 500 status codes weren't being caught by the try catch statement in `useFeedback.ts`. So to overcome that I return the response and then in `useFeedback.ts` throw an error if we get 400 or 500 status.